gtsam_unstable/slam/TransformBtwRobotsUnaryFactor.h

```diff
@@ -70,7 +70,7 @@ namespace gtsam {
     TransformBtwRobotsUnaryFactor(Key key, const VALUE& measured, Key keyA, Key keyB,
         const gtsam::Values& valA, const gtsam::Values& valB,
         const SharedGaussian& model) :
-          Base(cref_list_of<1>(key)), key_(key), measured_(measured), keyA_(keyA), keyB_(keyB),
+          Base(KeyVector{key}), key_(key), measured_(measured), keyA_(keyA), keyB_(keyB),
```

Comment:
It just dawned on me that maybe either `Factor` or `NonlinearFactor` should have an `initializer_list<Key>` constructor so that we can do `Base({key})` instead of `Base(KeyVector{key})`

This occurs several times in this PR

Reply:
2nd this but can be another PR

Reply:
I 3d this ! One of you wants to do it?